db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/ExtractOperatorNode.java

```diff
@@ -171,21 +171,21 @@ public ValueNode bindExpression(FromList fromList,
              && (extractField == DateTimeDataValue.SECOND_FIELD) ) {
             setType(new DataTypeDescriptor(
                             TypeId.getBuiltInTypeId(Types.DOUBLE),
-                            operand.getTypeServices().isNullable()
+                            getOperand().getTypeServices().isNullable()
                         )
                 );
         } else if (extractField == MONTHNAME_FIELD || extractField == DateTimeDataValue.WEEKDAYNAME_FIELD) {
             // name fields return varchar
             setType(new DataTypeDescriptor(
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
This seems to be a false positive